cudamapper/src/cudamapper_utils.cpp

```diff
@@ -65,5 +66,63 @@ void fuse_overlaps(std::vector<Overlap>& fused_overlaps, const std::vector<Overl
     set_relative_strand(fused_overlap);
     fused_overlaps.push_back(fused_overlap);
 }
+
+std::string string_slice(const std::string& s, std::size_t start, std::size_t end)
```

Comment:
Ok, now I see that this is a "local" function, i.e. it is not exposed in header. In that case you can move it to an anonymous namespace so it does not get exposed to linker

Reply:
Addressed in aac541d. Again, I learned something new here!